src/ui/qgsrendererv2propsdialogbase.ui

```diff
@@ -102,7 +102,7 @@
             <bool>false</bool>
            </property>
            <property name="collapsed" stdset="0">
-            <bool>false</bool>
+            <bool>true</bool>
```

Comment:
check a couple of lines below this one - the "saveCollapsedState" is the setting you want to change